meilisearch-release-changelog/v1.7.0.md

```diff
@@ -0,0 +1,255 @@
+
+# New features and improvements 🔥
+
+## Improved AI-powered search — Experimental
+
+To activate AI-powered search, set `vectorStore` to `true` in [the `/experimental-features` route](https://www.meilisearch.com/docs/reference/api/experimental_features). Consult the [Meilisearch documentation](https://www.meilisearch.com/docs/learn/experimental/vector_search#vector-search-with-auto-embeddings) for more information.
+
+🗣️ This is an experimental feature and we need your help to improve it! Share your thoughts and feedback on this [GitHub discussion](https://github.com/orgs/meilisearch/discussions/677).
+
+### New OpenAI embedding models
+
+When [configuring  OpenAI embedders](https://www.meilisearch.com/docs/learn/experimental/vector_search#generate-auto-embeddings-with-openai)), you can now specify two new models:
+- `text-embedding-3-small` with a default dimension of 1536.
+- `text-embedding-3-large` with a default dimension of 3072.
+
+These new models are cheaper and improve search result relevancy.
+
+### Custom OpenAI model dimensions
+
+You can configure `dimensions` for sources using the new OpenAI models: `text-embedding-3-small` and `text-embedding-3-large`. Dimensions must be bigger than 0 and smaller than the model size:
+
+```json5
+"embedders": {
+  "large": {
+    "source": "openAi",
+    "model": "text-embedding-3-large",
+    "dimensions": 512 // must be >0, must be <= 3072 for "text-embedding-3-large"
+  },
+  "small": {
+    "source": "openAi",
+    "model": "text-embedding-3-small",
+    "dimensions": 1024 // must be >0, must be <= 1536 for "text-embedding-3-small"
+  },
+  "legacy": {
+    "source": "openAi",
+    "model": "text-embedding-ada-002",
+    "dimensions": 1536 // must =1536  for "text-embedding-ada-002"
+  },
+  "omitted_dimensions": { // uses the default dimension
+    "source": "openAi",
+    "model": "text-embedding-ada-002",
+  }
+}
+```
+
+Done in #4375 by @Gosti.
+
+### GPU support when computing Hugging Face embeddings
+
+Activate CUDA to use Nvidia GPUs when computing Hugging Face embeddings. This can significantly improve embedding generation speeds.
+
+To enable GPU support through CUDA for HuggingFace embedding generation:
+
+
+1. [Install CUDA dependencies](https://huggingface.github.io/candle/guide/installation.html)
+2. Clone and compile Meilisearch with the `cuda` feature: `cargo build --release --package meilisearch --features cuda`
+3. Launch your freshly compiled Meilisearch binary
+4. [Activate vector search](https://www.meilisearch.com/docs/learn/experimental/vector_search#activate-vector-search)
+5. [Add a Hugging Face embedder](https://www.meilisearch.com/docs/learn/experimental/vector_search#generate-auto-embeddings-with-huggingface)
+
+Done by @dureuill in #4304.
+
```

Comment:
Additions related to [v1.7.0rc.2](https://github.com/meilisearch/meilisearch/releases/tag/v1.7.0-rc.2):
```suggestion

### Additional analytics related to the GPU support
* [Add a boolean analytics `gpu_enabled` when Meilisearch is compiled with CUDA](https://github.com/meilisearch/meilisearch/pull/4443) by @dureuill
```

Reply:
Nope, we don't add this kind of information to the changelogs. Not really user facing. The docs however should be updated.

Reply:
Yes, I was wondering about this one, I put it just in case

Reply:
And you were right, it's better to add it than forget it!